examples/platforms/simulation/radio.c

```diff
@@ -1321,3 +1326,37 @@ otError otPlatRadioGetRegion(otInstance *aInstance, uint16_t *aRegionCode)
 exit:
     return error;
 }
+
+otError ProcessNodeIdFilter(void *aContext, uint8_t aArgsLength, char *aArgs[])
+{
+    OT_UNUSED_VARIABLE(aContext);
+
+    otError error = OT_ERROR_NONE;
+
+    otEXPECT_ACTION(aArgsLength > 0, error = OT_ERROR_INVALID_ARGS);
+
+    if (!strcmp(aArgs[0], "clear"))
+    {
+        otEXPECT_ACTION(aArgsLength == 1, error = OT_ERROR_INVALID_ARGS);
+
+        memset(sDeniedNodeIds, 0, sizeof(sDeniedNodeIds));
+    }
+    else if (!strcmp(aArgs[0], "deny"))
+    {
+        otEXPECT_ACTION(aArgsLength == 2, error = OT_ERROR_INVALID_ARGS);
+
+        char *endptr;
+
+        uint16_t nodeId = (uint16_t)strtol(aArgs[1], &endptr, 0);
+        otEXPECT_ACTION(*endptr == '\0', error = OT_ERROR_INVALID_ARGS);
+        otEXPECT_ACTION(1 <= nodeId && nodeId <= MAX_NETWORK_SIZE, error = OT_ERROR_INVALID_ARGS);
+        sDeniedNodeIds[nodeId - 1] = true;
+    }
+    else
+    {
+        otEXPECT_ACTION(false, error = OT_ERROR_INVALID_ARGS);
```

Comment:
```suggestion
        otEXPECT_ACTION(false, error = OT_ERROR_INVALID_COMMAND);
```
`clear` and `deny` are more like sub-commands rather than arguments?